app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -1,11 +1,19 @@
 (() => {
   class ImageConstants {
-    
+
     get logos() {
       return {
         orange: "<%= asset_path("logo/logo_orange.png") %>"
       };
     }
+
+    get roles() {
+      return {
+        donor: "<%= asset_path("roles/donor.png") %>",
+        teacher: "<%= asset_path("roles/teacher.png") %>"
```

Comment:
i think the student.png is actually just clarinet.png, which ur homepage PR has. i'll wait for you to merge urs in first, and then fix this?